<launch>
  <group ns="test_controller" clear_params="true">
    <param name="type" type="string" value="CheckoutController" />
    <param name="timeout" type="double" value="30.0" />
  </group>

  <node pkg="qualification" type="visual_verifier.py" args="$(find qualification)/tests/laser_tilt_test/view.vcg" />
  
  <node pkg="qualification" type="robot_checkout.py" />
</launch>
